From: David McCullough <david.mccullough@accelecon.com>
Date: Thu, 21 Aug 2014 09:51:18 +1000
Subject: build: autoconf 2.63 bug workaround

Workaround for autoconf bug in 2.63 (on fireball).
The AS_IF macro does not generate valid shell if its run cmd is empty.
This is fixed in autoconf-2.65 but its easy to workaround for now once
you figure out the cause :-)
---
 m4/vapigen.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/vapigen.m4 b/m4/vapigen.m4
index 2c435e74..16ac3921 100644
--- a/m4/vapigen.m4
+++ b/m4/vapigen.m4
@@ -23,7 +23,7 @@ dnl Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  US
 # See http://live.gnome.org/Vala/UpstreamGuide for detailed documentation
 AC_DEFUN([VAPIGEN_CHECK],
 [
-  AS_IF([test "x$3" != "xyes"], [
+  AS_IF([test "x$3" != "xyes"],:;[
       m4_provide_if([GOBJECT_INTROSPECTION_CHECK], [], [
           m4_provide_if([GOBJECT_INTROSPECTION_REQUIRE], [], [
               AC_MSG_ERROR([[You must call GOBJECT_INTROSPECTION_CHECK or GOBJECT_INTROSPECTION_REQUIRE before using VAPIGEN_CHECK unless using the FOUND_INTROSPECTION argument is "yes"]])
